standard/links.lua

```diff
@@ -10,17 +10,31 @@ local _PREFIXES = {
     twitch4 = "https://www.twitch.tv/",
     twitch5 = "https://www.twitch.tv/",
     stream = "",
+    stream2 = "",
     mildom = "https://www.mildom.com/",
     huomaotv = "http://www.huomao.com/",
-    douyutv = "http://www.douyu.com/",
+    douyu = "http://www.douyu.com/",
     pandatv = "http://www.panda.tv/",
     huyatv = "http://www.huya.com/",
+    huyatv2 = "http://www.huya.com/",
+    toornament = "https://www.toornament.com/tournaments/",
+    toornament2 = "https://www.toornament.com/tournaments/",
+    toornament3 = "https://www.toornament.com/tournaments/",
+    battlefy = "https://www.battlefy.com/",
+    battlefy2 = "https://www.battlefy.com/",
+    battlefy3 = "https://www.battlefy.com/",
```

Comment:
Does #182 not fix this?

Reply:
problem is `steam` and `steam2`, they have different links
hence atm it only gets rid of the number for the icon and not for the link retrieval

Reply:
Shouldn't we change `steam2` to a different name then? That seems like it's just asking for trouble.

Reply:
do you mean like transform it into something different via `Links.transform` or like replace it in the wiki code (via bot runs on several wikis)?
idk how often `steam2` is used at all, on sc2 we do not have it, on RL i found no usage on main space, on CS 2 use cases on main space

Reply:
I think we should change the name here and bot the usages

Reply:
i dropped all the duplicates
for `steam2` i for now used `steamalternative`
if you have a better idea that would be good ...